R/junction-read-count.R

```diff
@@ -1,11 +1,12 @@
 #' Calculate the total number of junction reads overlapping with the introns of
```

Comment:
Rename variable names in function body and description:
   - `junctionFilePath` to `files`
   - `junctionType` to `fileType`
   - `junctionFileLabels` to `fileLabels`
   - `annotatedIntronRanges` to `intronRanges`
   - `promoterAnnotationData` to `promoterAnnotation`